pkg/workload/workload.go

```diff
@@ -240,15 +267,19 @@ func NewInfo(w *kueue.Workload, opts ...InfoOption) *Info {
 	for _, opt := range opts {
 		opt(&options)
 	}
-	info := &Info{
-		Obj: w,
-	}
+	info := &Info{Obj: w}
 	if w.Status.Admission != nil {
 		info.ClusterQueue = w.Status.Admission.ClusterQueue
 		info.TotalRequests = totalRequestsFromAdmission(w)
-	} else {
-		info.TotalRequests = totalRequestsFromPodSets(w, &options)
+		return info
```

Comment:
please don't modify the pre-existing code, unless really needed. It is not clear to me at all why the change. If needed, I would recommend a preparatory PR.